settings.js

```diff
@@ -97,5 +97,6 @@ window.spinnakerSettings = {
     // whether stages affecting infrastructure (like "Create Load Balancer") should be enabled or not
     infrastructureStages: process.env.INFRA_STAGES === 'enabled',
     jobs: false,
+    serialize: false,
```

Comment:
Feels like `serialization` might read better here. Or not. Now I've repeated both words so many times I don't think I have an opinion anymore.